indico_jacow/controllers.py

```diff
@@ -309,3 +313,102 @@ def _process(self, data):
         current_plugin.logger.info('Affiliation %r created by %r', aff, session.user)
         search_affiliations.bump_version()
         return AffiliationSchema().jsonify(aff)
+
+
+class BrevoAPIMixin:
+    @property
+    def api_instance(self):
+        if not hasattr(self, '_api_instance'):
+            from indico_jacow.plugin import JACOWPlugin
+            configuration_brevo = brevo_python.Configuration()
+            configuration_brevo.api_key['api-key'] = JACOWPlugin.settings.get('brevo_api_key')
+            self._api_instance = brevo_python.ContactsApi(brevo_python.ApiClient(configuration_brevo))
+        return self._api_instance
+
+    def get_contact_info(self, email):
+        try:
+            return self.api_instance.get_contact_info(email).to_dict()
+        except ApiException as e:
+            if e.status == 404:
+                return None
+            raise
+
+    def get_all_lists(self):
+        try:
+            return self.api_instance.get_lists().to_dict()
+        except ApiException as e:
+            raise IndicoError(f'Exception when retrieving Mailing Lists from Brevo: {e.reason}')
+
+    def create_contact(self, email, first_name, last_name, list_ids):
+        contact = brevo_python.CreateContact(
+            email=email,
+            attributes={'FIRSTNAME': first_name, 'LASTNAME': last_name},
+            list_ids=list_ids
+        )
+        return self.api_instance.create_contact(contact).to_dict()
+
+
+class RHMailingLists(RHUserBase, BrevoAPIMixin):
```

Comment:
```suggestion
class RHMailingLists(BrevoAPIMixin, RHUserBase):
```

(same for the others below)

The rightmost ones is the "main" base class, so IMHO it's more logical to have your main RH base class there, and then add mixins afterwards